src/screens/Settings/components/InfoScreen/styles.ts

```diff
@@ -0,0 +1,36 @@
+import { StyleSheet } from 'react-native';
+
+import { Colors } from '@UI/Colors';
+import { normalize } from '@UI/normalize';
+
+export const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    paddingVertical: 20,
+    paddingHorizontal: 15,
+    backgroundColor: Colors.background,
+  },
+  textContainer: {
+    marginBottom: 25,
+  },
+  text: {
+    fontFamily: 'Montserrat-Regular',
+    fontSize: 15,
+    color: Colors.text,
+  },
+  icon: {
+    width: normalize(30),
+    height: normalize(30),
+    alignSelf: 'flex-start',
+},
```

Comment:
the bracket is not properly aligned

+ add an eslint rule for checking/fixing this issue in future